OptimoveSDK/Sources/Classes/Optimobile/Optimobile.swift

```diff
@@ -89,12 +77,27 @@ class Optimobile {
         return instance != nil
     }
 
+    static var isSdkRunning: Bool {
+        return Optimobile.instance?.credentials != nil
+    }
+
     /**
          Initialize the Optimobile SDK.
      */
-    static func initialize(config: OptimobileConfig, initialVisitorId: String, initialUserId: String?) {
-        if instance !== nil {
-            assertionFailure("The OptimobileSDK has already been initialized")
+    static func initialize(config optimoveConfig: OptimoveConfig, initialVisitorId: String, initialUserId: String?) {
+        if instance !== nil, optimoveConfig.features.contains(.delayedConfiguration) {
+            completeDelayedConfiguration(config: optimoveConfig.optimobileConfig!)
+            return
+        }
+
+        guard instance == nil else {
+            Logger.error("The OptimobileSDK has already been initialized")
```

Comment:
throw here? previous intention was such

Reply:
I can throw an error here and then catch it in a caller, but there I will use the same logger. We should catch our errors and do not crash the host app.

Previous `assertionFailure` stops in debug only. Maybe we should do it additionally to logger.
https://developer.apple.com/documentation/swift/assertionfailure(_:file:line:)#discussion

Reply:
k, in android it also doesnt crash here, so, let's not. But i think crashing host app is sometimes fine/better than silently not doing what's expected

Reply:
let's keep `assertionFailure` here -- it will crash in the debug mode only. But keep a production version crash-free.